DependencyInjection/Configuration.php

```diff
@@ -38,10 +49,11 @@ public function getConfigTreeBuilder()
                             ->booleanNode('fullscreen')->defaultTrue()->end()
                             ->booleanNode('include_assets')->defaultTrue()->end()
                             ->scalarNode('tinymce_popup_path')->defaultValue('')->end()
-                            ->booleanNode('enableUserIntegration')->defaultFalse()->end()
+                            ->booleanNode('enable_user_integration')->defaultFalse()->end()
```

Comment:
will this break anything? (i've added a normalization function to still accept the old name, the result will now contain the new one)